docs/guides/developing/usage_examples.rst

```diff
@@ -323,3 +323,56 @@ This example is available as a working app in the OCIO source: src/apps/ociodisp
    oglApp->setShader(shaderDesc);
 
    oglApp->redisplay();
+
+Python
+++++++
+
+.. code-block:: python
+
+   import PyOpenColorIO as OCIO
+
+   # Step 1: Get the config
+   config = OCIO.GetCurrentConfig()
+
+   # Step 2: Lookup the display ColorSpace
+   device = config.getDefaultDisplayDeviceName()
+   transformName = config.getDefaultDisplayTransformName(device)
+   displayColorSpace = config.getDisplayColorSpaceName(device, transformName)
+
+   # Step 3: Create a DisplayTransform, and set the input and display ColorSpaces
+   # (This example assumes the input is scene linear. Adapt as needed.)
+
+   transform = OCIO.DisplayTransform()
```

Comment:
Here you'll want to use the `LegacyViewingPipeline` class, to which you will pass a `DisplayViewTransform` instance with the `display` and `view` requested above.

Reply:
Hey Michael, are you able to provide an example of what this would look like? I'm happy to make the change but I'm afraid I'm new to the API to contributing as part of Dev Days.

Reply:
Michael, don't worry about this, I'll work with Shane on this one.